src/relay/transforms/legalize.cc

```diff
@@ -35,14 +35,14 @@ namespace legalize {
 
 // Call registered FTVMLegalize of an op
 // Returns the legalized expression
-class Legalizer : public ExprMutator {
+class Legalizer : public ExprRewriter {
  public:
   explicit Legalizer(const std::string& legalize_map_attr_name)
       : legalize_map_attr_name_{legalize_map_attr_name} {}
 
-  Expr VisitExpr_(const CallNode* call_node) {
+  Expr Rewrite_(const CallNode* call_node, const Expr& post) override {
     // Get the new_call node without any changes to current call node.
-    Expr new_e = ExprMutator::VisitExpr_(call_node);
+    Expr new_e = post;
```

Comment:
Can we just rename `post` to `new_e` and remove this line?

Reply:
I could not directly rename, as `new_e` was supposed to be the new expr and `post` is a cont. But, I have cleaned up the code, so that `new_e` is not needed anymore.